ansible/roles/uan_interfaces/tasks/main.yml

```diff
@@ -276,4 +342,5 @@
       - "'Compute' not in group_names"
 
   when:
+    - not cray_cfs_image
```

Comment:
This is controlled at the playbook level so I don't think we need this check.

Reply:
Fixed